### DISCUSSION TITLE
Discovery Session: Kick Off (#1)

### DISCUSSION THREAD

**Discussion by Cozonacci** [OrgTBoot/trading-boot#21] [Show and tell]
@vladislavmunteanu @maimas @iuter777 @Cozonacci 
Putem capta aici subiecte pentru sesiunea de vineri

**Comment by maimas** [comment]
Hight level agenda:
- how do we bocome milioners?
- - how to minimize risks !!! This is paramount!
- - combo orders (when and how)
- - when to take profits, should we intervene in automated routines?
- - should we consider improving the quote change ingestion process to reduce lag (subscription model), now vs latter.
- - screening, what can we do better?
